packages/11ty/_layouts/objects-catalog.webc

```diff
@@ -0,0 +1,8 @@
+---
+layout: base.11ty.js
+---
+<page-header></page-header>
+<style @raw="getBundle('css')" webc:keep></style>
+<!-- note: naming this `<objects-catalog-content>` prevented `<objects-catalog>` from rendering -->
```

Comment:
calling out this HTML comment here as it will probably get removed: naming the content wrapper component `<objects-catalog-content>` (instead of `<catalog-content>`) prevented `<objects-catalog>` from rendering at all